### PR TITLE
Fix Possum branch CI build

### LINKED ISSUE #181: Fix possum branch CI build

### PR DESCRIPTION
**Pull request by jtuttle**
Closes #181.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ COPY conjur-cli.gemspec conjur-cli.gemspec
 COPY lib/conjur/version.rb lib/conjur/version.rb
 
 # Make sure only one version of bundler is available
-RUN gem uninstall bundler -i /usr/local/lib/ruby/gems/${RUBY_VERSION}.0 bundler || true && \
-  gem uninstall bundler -aIx && \
+RUN gem uninstall bundler -i /usr/local/lib/ruby/gems/${RUBY_VERSION} bundler || true && \
```
Comment (dustinmm80) [inline Dockerfile]: why do we have to run `gem uninstall bundler` twice here?

File: Gemfile [hunk 2]
```diff
@@ -8,4 +8,4 @@ gemspec
 
 gem 'activesupport', '~> 4.2'
 
-gem 'conjur-api', git: 'https://github.com/conjurinc/api-ruby.git', branch: 'possum'
+gem 'conjur-api', '~> 5.0.0'
```
Comment (dustinmm80) [inline Gemfile]: You can shorten this to `'~> 5`

File: Jenkinsfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ pipeline {
   stages {
     stage('Test 2.2') {
       environment {
-        RUBY_VERSION = '2.2'
+        RUBY_VERSION = '2.2.0'
```
Comment (dustinmm80) [inline Jenkinsfile]: instead of 0s - often buggy releases - we should use the latest `y` versions, for ruby versions `2.x.y`. This ensures that we're running against recent Rubies

File: Jenkinsfile [hunk 4]
```diff
@@ -9,7 +9,7 @@ pipeline {
   stages {
     stage('Test 2.2') {
```
Comment (dustinmm80) [inline Jenkinsfile]: We should run these stages in parallel. I can add a commit to this PR showing how.

File: Dockerfile (hunk 1 above)
Comment (jtuttle) [reply Dockerfile]: Not sure, but it looks like it's once for the currently selected gemset and once for the version-specific gemset. No idea why this is necessary. Could try without the latter, perhaps.

File: Jenkinsfile (hunk 3 above)
Comment (jtuttle) [reply Jenkinsfile]: Changed to most recent `2.x` version.
